aesm-client/src/lib.rs

```diff
@@ -192,6 +212,7 @@ impl AesmClient {
         )
     }
 
+    #[cfg(not(target_env = "sgx"))]
     pub fn get_launch_token(
```

Comment:
as of now, the crate doesnt compile for unix (in master branch) without using the `feature="sgxs"` flag.
Is that a something we want?
I believe one solution would be to add a `#[cfg(feature = "sgxs")]` for `get_launch_token`

Reply:
Oops. That's probably an oversight. Please add the feature yes